### PR TITLE
Update application.scss to fix search bar out of place in schedule, rooms as well as tracks page

### LINKED ISSUE #2214: Generated Page: Search bar out of place
The search bar is out of place on generated pages. See below. 

Compare https://summit.fossasia.org/event/tracks.html

Screenshot from 2020-03-02 13-12-29

### PR DESCRIPTION
**Pull request by rishi-raj-jain**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

[Performed as a part of Codeheat]

#### Short description of what this resolves:
This PR fixes the issue 'Generated Page: Search bar out of place' #2214 by changing the width from 100 to 98 of the two classes(only where it was required), as 100% forces the element to take the full width hence go out of place, while 98% keeps the element take up the maximum space inline as well as seems fine in responsive sizes. Also, the search icon seem to be going around the corner and looked out of sync with the Search placeholder, hence shifted it bit to the right. This definitely makes it go easier on the eyes (Right use of the whitespace)

#### Changes proposed in this pull request:
- Fix search bar width in schedule, tracks and rooms page,
- Align position of search icon to utilize whitespace

Screenshots for the new UI:
Screenshot (483)
Screenshot (484)
Screenshot (485)

Fixes #2214

### CODE REVIEW

File: src/backend/_scss/application.scss [hunk 1]
```diff
@@ -1672,7 +1672,7 @@ a {
 
   input {
     line-height: 30px;
-    width: 100%;
+    width: 98%;
```
Comment (iamareebjamal) [inline src/backend/_scss/application.scss:1675]: This feels like a workaround. Please tweak the paddings, but this and above width should be 100%

Tomorrow, something else might change layout and someone might have to change the width to 96 or 95%

It is not a sustainable solution. Markup must be fixed

**Review by vsasvipul0605 — APPROVED** [review]

**Review by vsasvipul0605 — APPROVED** [review]

**Review by Raunaksingh100 — APPROVED** [review]

**Comment by iamareebjamal** [comment]
There is literally a request for change above your approvals. Don't know why you are approving

File: src/backend/_scss/application.scss (hunk 1 above)
Comment (Subham142) [reply src/backend/_scss/application.scss:1675]: if we could make the position of the div  absolute and change according to that ,also taking care of the responsiveness 

is this the right approch?

**Comment by rishi-raj-jain** [comment]
Won't be able to take up this further.
Anybody else feel free to pick up.

**Comment by djcruz93** [comment]
I am working on this issue.

**Pull request closed without merging**